recipes/mimalloc/all/test_package/conanfile.py

```diff
@@ -62,6 +62,8 @@ def test(self):
         if tools.cross_building(self.settings):
             return
 
+        self.output.info(f"Environment append: {self._environment}")
```

Comment:
Conan needs to support the older syntax still

```suggestion
        self.output.info("Environment append: {}".format(self._environment))
```